Remotion/Web/Test.Shared/MultiplePostBackCatching/TestControlGenerator.cs

```diff
@@ -245,7 +248,7 @@ private Image CreateImage (string prefix, string text)
       Image image = new Image ();
       image.ID = CreateID (prefix, "Inner");
       image.AlternateText = text;
-      image.ImageUrl = "~/Images/Image.gif";
+      image.ImageUrl = _resourceUrlFactory.CreateRelativeResourceUrl ("Images/Image.gif");
```

Comment:
Since we're touching the world anyway, please rename `Images` folder to `Image` and use `ResourceType.Image`.